android/app/src/main/java/host/exp/exponent/MainApplication.java

```diff
@@ -48,7 +48,7 @@
 import expo.modules.sqlite.SQLitePackage;
 import expo.modules.taskManager.TaskManagerPackage;
 import expo.modules.webbrowser.WebBrowserPackage;
-import expolib_v1.okhttp3.OkHttpClient;
+import okhttp3.OkHttpClient;
```

Comment:
Add a breaking change to CHANGELOG that people we removed use of `expolib_v1.okhttp` in favor of `okhttp` (this means people will need to edit their `MainApplication` on their own when upgrading)